shell/platform/embedder/embedder.h

```diff
@@ -388,6 +387,10 @@ typedef void (*FlutterPlatformMessageCallback)(
     const FlutterPlatformMessage* /* message*/,
     void* /* user data */);
 
+typedef void (*FlutterDataCallback)(const void* /* data */,
```

Comment:
Missed this before: why is data a `void*` rather than a `uint8_t*` like the data in a message and the data in a response going the other way? It seems like an odd/confusing mismatch.